persistence/test/actor_test.go

```diff
@@ -37,3 +40,79 @@ func TestGetAllStakedActors(t *testing.T) {
 	require.Equal(t, genesisStateNumApplications, actualApplications)
 	require.Equal(t, genesisStateNumFishermen, actualFishermen)
 }
+
+func TestGetValidatorSet_Hash(t *testing.T) {
+	expectedHashes := []string{
+		"5831e3e6a8d3beda0adb6027126a4bc3b0181836eebcc45a83dc2970ee9b4468",
+		"1f6faa8782a4608341fef83ee72c9e9cd3f96e042f2ddfdeebda8d971bb2ac13",
+	}
+
+	db := NewTestPostgresContext(t, 0)
+	currValSet, err := db.GetValidatorSet(0)
+	require.NoError(t, err)
+	currValSetHash := hashValSet(t, currValSet)
+	require.Equal(t, expectedHashes[0], currValSetHash)
+
+	currHeight := int64(1)
+	db.Height = currHeight
+
+	currValSet, err = db.GetValidatorSet(currHeight - 1)
+	require.NoError(t, err)
+	currValSetHash = hashValSet(t, currValSet)
+	nextValSet, err := db.GetValidatorSet(currHeight)
+	require.NoError(t, err)
+	nextValSetHash := hashValSet(t, nextValSet)
+
+	require.Equal(t, expectedHashes[0], currValSetHash)
```

Comment:
Why are we checking `expectedHashes[0]` in both cases?

Reply:
added comments that should clarify this